2-mandatory/2-financial-times.js

```diff
@@ -23,59 +40,83 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  // TODO
+  const headlinesWithNumbersArray = [];
+
+  for (const eachTitle of allArticleTitles) {
+    for (const char of eachTitle) {
+      if (!isNaN(parseInt(char))) {
```

Comment:
that's intrusting use of the Boolean values i learned something here thank you.

Reply:
Hi Nasir :)   I like this too, took me half a day to find this and sink in,     especially this 3 devils :)))  ===>      !  is NaN   
this feels like a maze inside another maze